storage/ledger/benchmark/benchmark_test.go

```diff
@@ -5,107 +5,95 @@ package benchmark
 // 	StorageBenchmark()
 // }
 
-import (
-	"io/ioutil"
-	"math/rand"
-	"os"
-	"testing"
-	"time"
-
-	"github.com/onflow/flow-go/module/metrics"
-	"github.com/onflow/flow-go/storage/ledger"
-	"github.com/onflow/flow-go/storage/ledger/ptrie"
-	"github.com/onflow/flow-go/storage/ledger/utils"
-)
-
-func BenchmarkStorage(b *testing.B) { benchmarkStorage(100, b) }
+// func BenchmarkStorage(b *testing.B) { benchmarkStorage(100, b) }
 
 // BenchmarkStorage benchmarks the performance of the storage layer
```

Comment:
should we add a todo here? Or just delete?

Reply:
Its an old ledger, this benchmark can go away.